R/db_oracle.R

```diff
@@ -1,4 +1,3 @@
-#' @rdname hidden_aliases
```

Comment:
The method does need to be documented, so having this here is fine, you just need to add the hidden.R file as well. You can copy the one from DBI (https://github.com/cran/DBI/blob/882c3921ee58d1fa97298697bf2a50161a8e401a/R/hidden.R#L1-L8)